algorithms/kernel/linear_regression/oneapi/linear_regression_train_kernel_oneapi.h

```diff
@@ -68,6 +67,22 @@ class BatchKernelOneAPI<algorithmFPType, training::normEqDense> : public daal::a
                              bool interceptFlag) const;
 };
 
+template <typename algorithmFPType, training::Method method>
+class OnlineKernelOneAPI
+{};
+
+template <typename algorithmFPType>
+class OnlineKernelOneAPI<algorithmFPType, training::normEqDense> : public daal::algorithms::Kernel
```

Comment:
In some other algorithms, kernel name includes brief algorithm name. Is it in accordance with our naming policy?

Reply:
This kernel is placed in namespace of algorithm. With this approach collisions will not occur.